### PR TITLE
Update run-koajs-on-google-app-engine.md

### PR DESCRIPTION
**Pull request by JumpLink**
The old server.js code snippet is not working with the current koa.js version, this pr fix that

### CODE REVIEW

**Comment by JumpLink** [comment]
I signed it!

File: tutorials/run-koajs-on-google-app-engine.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ Platform.
...
 
-    const koa = require('koa');
-    const app = koa();
+    const Koa = require('koa');
+    const app = new Koa();
 
-    app.use(function *() {
-      this.body = 'Hello World!';
+    app.use(ctx => {
+      ctx.body = 'Hello World';
```
Comment (hegemonic) [inline tutorials/run-koajs-on-google-app-engine.md]: Please restore the exclamation point so that the text that follows the code sample is accurate.

Comment (jmdobry) [inline tutorials/run-koajs-on-google-app-engine.md]: style nit:

s/`ctx =>`/`(ctx) =>`/

Comment (JumpLink) [inline tutorials/run-koajs-on-google-app-engine.md]: Should be correct now

**Pull request merged**